spec/decorators/event_decorator_spec.rb

```diff
@@ -12,5 +12,5 @@
   it { expect(subject.local_map_link).to eq("<a target=\"_blank\" href=\"https://www.google.com.br/maps/search/address\"><img class=\"img-responsive pull-right\" src=\"https://maps.googleapis.com/maps/api/staticmap?center=address&amp;zoom=15&amp;size=300x300&amp;markers=color:0xCC342D|address\" alt=\"Staticmap?center=address&amp;zoom=15&amp;size=300x300&amp;markers=color:0xcc342d|address\" /></a>") }
   it { expect(subject.summary).to        eq("<p>summary</p>\n") }
   it { expect(subject.description).to    eq("<p>description</p>\n") }
-  it { expect(subject.details_link).to   eq("<a class=\"btn btn-success\" href=\"/events/event\">Maiores informações</a>") }
+  it { expect(subject.details_link).to   eq("<a class=\"btn-more-details\" href=\"/events/event\">Maiores informações</a>") }
```

Comment:
Line is too long. [125/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.